tests/commands/pipeline/test_run.py

```diff
@@ -48,6 +60,12 @@ def test_match_pipeline_ambiguous(runner, logged_in_and_linked):
         match_pipeline(config, 'Train')
 
 
-def add_valid_pipeline_yaml():
-    with open(get_project().get_config_filename(), 'w') as yaml_fp:
+def add_valid_pipeline_yaml(custom_yaml=None):
+    project = get_project()
+    if custom_yaml:
+        directory = project.directory
+        config_filename = os.path.join(directory, custom_yaml)
+    else:
+        config_filename = project.get_config_filename()
```

Comment:
Correct me if I'm wrong, but isn't this...
```suggestion
    config_filename = project.get_config_filename(custom_yaml)
```